core/base/src/main/java/alluxio/security/authorization/DefaultAccessControlList.java

```diff
@@ -166,15 +166,16 @@ public void setEntry(AclEntry entry) {
   }
 
   /**
-   * Returns true if the default ACL is empty.
-   * @return true if empty
+   * @return whether this access-control list is empty
+   *         ({@code true}) or not ({@code false})
```

Comment:
Thes second line is redundant as the return type is a `boolean`, so it's either true of false.